docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -14,7 +14,7 @@ The `Estimator` primitive can compute an expectation value, and the `Sampler` pr
 Follow these instructions to get the expected value of an observable for a given quantum circuit with the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) primitive.
 
 <Admonition type="note">
-  While this guide uses Qiskit’s reference implementation, the `Estimator` primitive can be run with any provider using [`qiskit.primitives.BackendEstimator`](../api/qiskit/qiskit.primitives.BackendEstimator).
+  While this guide uses the Qiskit reference implementation, the `Estimator` primitive can be run with any provider using [`qiskit.primitives.BackendEstimator`](../api/qiskit/qiskit.primitives.BackendEstimator).
```

Comment:
This should be linked to the EstimatorV2 in `qiskit-ibm-runtime` I think. At the moment, this is only available in a feature branch, so I'm not sure how the linking should be done right now. Maybe point to that project as a whole?

Reply:
Removed reference to backend estimator in [d7c0f74](https://github.com/Qiskit/documentation/pull/812/commits/d7c0f740f42d07ba267dddc41963cb9ff4c2b946) after discussion to create a new topic for backend primitives.